### PR TITLE
Added gtag4 script

### PR DESCRIPTION
**Pull request by amityt**

### CODE REVIEW

File: app/client/public/index.html [hunk 1]
```diff
@@ -67,6 +67,16 @@
 			gtag("config", "UA-92076314-27");
 		</script>
 
+		<!-- Google tag (gtag.js) -->
```
Comment (gdsoumya) [inline app/client/public/index.html:70]: There's already a tag existing do we need both of them or can we remove the old one? Also the ci build is failing.

**Review by gdsoumya — CHANGES_REQUESTED** [review]

File: app/client/public/index.html (hunk 1 above)
Comment (amityt) [reply app/client/public/index.html:70]: Yes, we have have the dual support for this. Once the older is depreciated, we can remove it.

Comment (amityt) [reply app/client/public/index.html:70]: Looking in the build issue. Thanks @gdsoumya

**Comment by amityt** [comment]
Closing this PR. Adding changes in this https://github.com/litmuschaos/charthub.litmuschaos.io/pull/355

**Pull request closed without merging**